src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Component Reference
```

Comment:
It should be in plural, right?
```suggestion
title: Components Reference
```

Reply:
Maybe even "Built-in Components Reference", since it's not a reference for generic components?